app/javascript/vue/global_search/groups/folders.vue

```diff
@@ -1,14 +1,55 @@
 <template>
-  <div class="bg-white rounded p-4 mb-4">
+  <div v-if="total" class="bg-white rounded p-4 mb-4">
     <h2 class="flex items-center gap-2 mt-0 mb-4">
       <i class="sn-icon sn-icon-folder"></i>
       {{ i18n.t('search.index.folders') }}
+      [{{ total }}]
     </h2>
+    <div>
+      <div class="grid grid-cols-[auto_auto_auto_auto] items-center">
```

Comment:
For code colum you can add pixel definition so it will not grow too big on wide screens

Reply:
We do not have code breadcrumb for Project folders at the moment